core/commands/add.go

```diff
@@ -30,9 +30,11 @@ const (
 
 var AddCmd = &cmds.Command{
 	Helptext: cmds.HelpText{
-		Tagline: "Add a file or directory to ipfs.",
+		Tagline: "Add a file to ipfs.",
```

Comment:
I think this is worse than it was before. `ipfs add` is for adding both files and directories, and the command summary line should say so. The description can be more clear, that's separate.